lib/lrama/grammar/parameterizing_rule_builder.rb

```diff
@@ -10,14 +10,12 @@ def initialize(name, parameters, rhs)
         @required_parameters_count = parameters.count
       end
 
-      def build_rules(token, rule_counter, lhs_tag, line)
+      def build_rules(token, actual_args, rule_counter, lhs_tag, line, rule_builders)
```

Comment:
[note] I agree to not giving `actual_args` default value because how to construct argument map depends on the type of caller.